source/codegen/metadata/nixnet/functions.py

```diff
@@ -507,6 +508,7 @@
             },
             {
                 'direction': 'in',
+                'include_in_proto': False,
```

Comment:
[Minor] For accuracy, can you update `attributeText`'s direction to 'out' on line 516. 

I think after you update it you'll have to either add size mechanism (you could use passed-in with value 'attributeTextSize') or add a size suppression to functions_addon.py like what's done for xnet sockets: https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/nixnetsocket/functions_addon.py.

Reply:
Done. Thanks for noticing this.